Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1/types.go

```diff
@@ -2597,7 +2597,14 @@ type SecurityContextConstraints struct {
 
 	// AllowPrivilegedContainer determines if a container can request to be run as privileged.
 	AllowPrivilegedContainer bool `json:"allowPrivilegedContainer" description:"allow containers to run as privileged"`
+	// Required capabilities are the capability set that must be set or defaulted into
+	// the containers.  Any capability listed in add will be added to the container even if
+	// it is not requested.  Any capability listed in drop will be removed from the container
+	// even if it is requested to be added.
```

Comment:
If someone requests a capability to be added explicitly, and it is in the drop list for required capabilities, I would expect the pod to get rejected with a failure, not accepted with the capability dropped.